### ISSUE TITLE
decompression library not compiled

### ISSUE THREAD

**Issue by ydehond6** [NKI-AI/xdrt#6]
**Describe the bug**
Used xdr2img to convert a .scan file to .nrrd file.  Error message (decompression library not compiled) after changing in function xdr_reader.py r.21 to nki_compression = ctypes.cdll.LoadLibrary(
        str(Path(path.dirname(path.abspath(__file__))).parent / "lib" / nki_decompress_lib  # type: ignore
    ))

**Environment**
dlup version:
How installed: pip install xdrt
Python version: 3.8
Operating System: windows 10

**Comment by jonasteuwen** [comment]
Thanks. I have pushed an update to the windows_fix branch. I have installed both MSVC and CMake. Will double check later today if it runs on windows properly now (it does on OS X).

**Comment by jonasteuwen** [comment]
The version updated on PyPi should work now.  I have used Cmake (https://cmake.org/download/) for windows and MSVC as compiler. If you do that `pip install xdrt` should work if cmake is in your path. Let me know if you can get it to work, I will write some documentation about it.

After I'll work on #7.

**Comment by ydehond6** [comment]
Thank you for updating the code. However, I have problems with installing xdrt now.  
subprocess.CalledProcessError: Command '['cmake', '--version']' returned non-zero exit status 1.
I have  b'cmake version 3.20.5\n\nCMake suite maintained and supported by Kitware (kitware.com/cmake).\n'

**Comment by jonasteuwen** [comment]
CMake does not appear to be in the path of your environment, the complaint is that it cannot run cmake —version

**Comment by ydehond6** [comment]
Thank you! it worked by using a new enviroment and not install cmake through pip before installation of xdrt.

**Comment by jonasteuwen** [comment]
Ah, that's great. Can you summarize your steps?

Now you should be able to do `xdr2img <object with .scan or .xdr extension> output.nii.gz` and open `output.nii.gz` in ITK-Snap or Slicer or so. I'll work on your other request as well.

**Comment by ydehond6** [comment]
install cmake at https://cmake.org/download/
in anaconda powershell prompt : 
>> conda create -n test python==3.8
>> conda activate test
>> pip install xdrt

and :  xdr3img input.scan output.nrrd  worked

So I don't know why it first did not work but now it works
